Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs

```diff
@@ -3652,11 +3697,16 @@ private Size CalculateAvailableCalendarViewSize(Size availableSize)
             availableSize.Width -= this.BorderThickness.Left + this.BorderThickness.Right + this.Padding.Left + this.Padding.Right;
             availableSize.Height -= this.BorderThickness.Top + this.BorderThickness.Bottom + this.Padding.Top + this.Padding.Bottom;
 
-            if (this.navigationPanel != null)
+            if (this.navigationPanel != null && this.HeaderVisibility == Visibility.Visible)
             {
                 availableSize.Height -= this.navigationPanel.ActualHeight;
             }
 
+            if (this.footerPanel != null && this.FooterVisibility == Visibility.Visible)

```

Comment:
When the visibility is changed dynamically are the measure and arrange phase triggered?

Reply:
yep